yandextank/plugins/JMeter/plugin.py

```diff
@@ -38,6 +42,8 @@ def __init__(self, core, cfg, cfg_updater):
         self.jmeter_log = None
         self.start_time = time.time()
         self.jmeter_buffer_size = None
+        self.jmeter_udp_addr = None
+        self.shutdown_timeout = 10
```

Comment:
add to plugin's config schema (https://github.com/yandex/yandex-tank/blob/develop/yandextank/plugins/JMeter/config/schema.yaml). Default values are also set through schema.